src/views/asset-edit-view/asset-edit-view.tsx

```diff
@@ -44,7 +49,7 @@ export const AssetEditView = (): JSX.Element => {
   return (
     <>
       {asset.assetType === "Dwelling" || asset.assetType === "LettableNonDwelling" ? (
-        <AssetEditLayout assetDetails={asset} />
+        <AssetEditLayout assetDetails={asset} tenureApiObject={tenure} />
```

Comment:
Unless you're precious about the variable name, I would suggest renaming `tenureApiObject` to something like `assetTenureDetails`

Reply:
there's already an assetTenureDetails (the tenure details inside the asset object)

Reply:
We need to differentiate between a couple of things:
1. The Asset object from the asset-api
2. The tenure details within the asset object
3. The Tenure from the tenure-api
4. The tenuredAsset within the tenure object

When we say assetTenureDetails, are we referring to [2] or [4] ?

Reply:
Correct, in my head I was thinking `assetTenureDetails` should be what you get back when you call the tenure API, so the Tenure object (#2). As in the Tenure for a given Asset.

You're right Asset has a `tenure` property but this doesn't have as much information within, unlike the Tenure obj you get from the Tenure API.

This isn't a big deal anyway, even if it could cause confusion at first, so you can decide what you think it's best